nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# Ticket number23

Fixes:
- Users, when viewing a product's details page, can like/dislike the product.
- When the user views the details page again, the page will display if the user liked or disliked the product

Changes proposed in this request & reasons behind organizational or architectural decisions:
-Created a Product Detail View Model and User Product Like view model

Steps to test:
- Start Bangazon
- Visit the details page of a product
- Select Like
- See that the count of Likes by all users has gone up by one and that the option to like/dislike is replaced by a message
- Look at the details for another product
- Choose dislike
- See that the buttons have gone away and that a message is displayed that you dislike the item

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
- N/A

Commit message:
- users can like/dislike product and when they view that product detail again the product will reflect whether the user chose like or dislike
--
Link to feature ticket:
-https://github.com/nss-day-cohort-30/bangazon-site-mighty-mahi-mahi/issues/23

@mighty-mahi-mahi
